src/primitives/polyline/styled.rs

```diff
@@ -119,6 +158,8 @@ where
                         .map(|point| Pixel(point, stroke_color)),
                 ),
                 _ => {
+                    let mut display = display.translated(self.primitive.translate);
```

Comment:
There might be a little performance improvement if this would only use a translated draw target if `translate != Point::zero()`.

Reply:
Good shout, however the only way I could get this to work was by copying and pasting most of the block:

```rust
if self.primitive.translate != Point::zero() {
    let mut display = display.translated(self.primitive.translate);

    for line in ScanlineIterator::new(self) {
        let rect = line.to_rectangle();

        if !rect.is_zero_sized() {
            display.fill_solid(&rect, stroke_color)?;
        }
    }
} else {
    for line in ScanlineIterator::new(self) {
        let rect = line.to_rectangle();

        if !rect.is_zero_sized() {
            display.fill_solid(&rect, stroke_color)?;
        }
    }
}
```

Any ideas on deduping this?

Reply:
Move the duplicate part into another method that is generic over any `DrawTarget`. You will probably need to add a new `impl<C> Styled<Polyline<'_>, PrimitiveStyle<C>>`  block, if it doesn't already exist.

Reply:
Was hoping there's a better way but ah well :). I've deduped it now.